tests/31sync/05presence.pl

```diff
@@ -34,6 +34,8 @@
 test "User is offline if they set_presence=offline in their sync",
    requires => [ local_user_fixture( with_events => 0 ),
                  qw( can_sync ) ],
+   # This test passes if we set a displayname for the user.
+   expect_fail => 1,
```

Comment:
Hmm. Can we raise a bug (SYN or SYT) for this failure, and change it to `bug => "BUGNAME"`? That indicates more accurately what the failure actually is.